izanami-frontend/src/components/SearchComponant/SearchResults.tsx

```diff
@@ -0,0 +1,191 @@
+    )
+    .slice(-5);
+
+  return (
+    <div className="search-container">
+      {filteredHistory.length > 0 && resultStatus.state == "INITIAL" && (
+        <div className="search-result">
+          <>
+            <span>Recent</span>
+            <ul className="search-ul nav flex-column">
+              {filteredHistory.map((term, index) => (
+                <li className="search-ul-item" key={index}>
+                  <Link
+                    to={getLinkPath(term) || "#"}
+                    onClick={() => handleItemClick(term)}
+                  >
+                    <ul
+                      className="breadcrumb"
+                      style={{
+                        marginBottom: "0rem",
+                      }}
+                    >
+                      {modalStatus.all &&
+                        term.path.map((pathElement, pathIndex) => (
+                          <li
+                            className="breadcrumb-item"
+                            key={`${term.name}-${pathIndex}`}
+                          >
+                            {typeDisplayInformation
+                              .get(pathElement.type)
+                              ?.icon() ?? ""}
+                            {pathElement.name}
+                          </li>
+                        ))}
+                      <li className="breadcrumb-item" key={`${term.name}-name`}>
+                        {typeDisplayInformation.get(term.type)?.icon() ?? ""}
+                        {term.name}
+                      </li>
+                    </ul>
+                  </Link>
+                </li>
+              ))}
+            </ul>
+          </>
+        </div>
+      )}
+
+      {resultStatus.state !== "INITIAL" && (
+        <div className="search-result">
+          {resultStatus.state === "ERROR" ? (
+            <div>{resultStatus.error}</div>
+          ) : resultStatus.state === "PENDING" ? (
+            <div>Searching...</div>
+          ) : (
+            resultStatus.state === "SUCCESS" &&
+            (groupedItems?.size > 0 ? (
+              <ul className="search-ul nav flex-column">
+                {[...groupedItems.keys()].map(
+                  (originTable, originTableIndex) => (
+                    <li className="search-ul-item" key={originTable}>
+                      <span>
+                        {typeDisplayInformation.get(originTable)?.displayName ??
+                          ""}
+                      </span>
+                      {groupedItems.get(originTable)?.map((item, index) => (
+                        <ol
+                          className="search-ul nav flex-column"
+                          key={`${item.name}-${originTableIndex}-${index}`}
+                        >
+                          <li
+                            className="search-ul-item"
+                            key={`${item.name}-${originTable}-${index}`}
```

Comment:
I don't think this item needs a key, only "root" item of iteration (here <ol>) need to have one.